pay-api/src/pay_api/resources/v1/eft_short_names.py

```diff
@@ -73,6 +74,36 @@ def get_eft_shortnames():
     return jsonify(response), status
 
 
+@bp.route('/summaries', methods=['GET', 'OPTIONS'])
+@cross_origin(origins='*', methods=['GET'])
+@_jwt.requires_auth
+@_jwt.has_one_of_roles([Role.SYSTEM.value, Role.MANAGE_EFT.value])
+def get_eft_shortname_summaries():
+    """Get all eft short name summaries."""
+    current_app.logger.info('<get_eft_shortname_summaries')
+    page: int = int(request.args.get('page', '1'))
+    limit: int = int(request.args.get('limit', '10'))
+    short_name = request.args.get('shortName', None)
```

Comment:
perhaps candidate for a function + tuple return?
`page, limit, short_name = get_request_args(['page','limit','shortName'])`

Reply:
Will ponder this one a bit more, it starts getting clunky when we consider defaults along with a longer list of params.